### PR TITLE
fixed according phpv8.0

### PR DESCRIPTION
**Pull request by ankitdas13**
issue : https://github.com/razorpay/razorpay-php/issues/296

### CODE REVIEW

File: src/Webhook.php [hunk 1]
```diff
@@ -28,12 +28,12 @@ public function all($options = array())
      *
      * @param array $attributes
      *
-     * @return Webhooks
+     * @return Webhook
```
Comment (ndhaka007) [inline src/Webhook.php:31]: Also add `@param string $id`

File: src/Webhook.php [hunk 2]
```diff
@@ -28,12 +28,12 @@ public function all($options = array())
...
      */
-    public function edit($attributes = array(), $id)
+    public function edit($id, array $attributes = array())
     {
         $url = $this->getEntityUrl() . $id;
 
-        return $this->request(Requests::PUT, $url, $attributes);
+        return $this->request('PUT', $url, $attributes);
```
Comment (ndhaka007) [inline src/Webhook.php:37]: Is this constant creating any issue?

File: src/Webhook.php (hunk 1 above)
Comment (ankitdas13) [reply src/Webhook.php:31]: doc comment is updated.

File: src/Webhook.php (hunk 2 above)
Comment (ankitdas13) [reply src/Webhook.php]: code has been reverted

**Review by ndhaka007 — APPROVED** [review]
LGTM

File: src/Webhook.php [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?php
 
 namespace Razorpay\Api;
-use Requests;
+
```
[the remark this answers is not in the document]
Comment (ankitdas13) [reply src/Webhook.php]: yes the issue is only about optional parameter . i have reverted the code

**Pull request merged**